modules/client/src/addresslistVoting/types.ts

```diff
@@ -37,3 +38,12 @@ export type AddresslistVotingPluginPrepareInstallationParams = {
   daoAddressOrEns: string;
   versionTag?: VersionTag;
 };
+
+export type AddresslistVotingPluginPrepareUpdateParams = {
+  pluginAddress: string;
+  daoAddressOrEns: string;
+  newVersion: VersionTag;
+  updateParams?: any;
```

Comment:
couldn't we use `Object` or `any[]`. I assume by the name that it has to be either an object or an array